section2/reflection.md

```diff
@@ -1,29 +1,57 @@
 ## Section 2 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+* I find that I am great at asking questions when I am in the same room as people, but I am pretty bad at it when not in the same room (this goes for virtual rooms as well). I am very self-sufficient, and don't like putting a burden on people, so I will pretty much always try to figure it out on my own first. I definitely tend to take too long to ask questions.
```

Comment:
I understand this, I typically like to solve things for myself. However, I would say that getting in the habit of error documenting is a very valuable skill that not every developer hones. I suggest:
1. Start a slack message or a gist or digital notes page
2. Describe the issue/error
3. Include screenshots and code snippets
4. Start researching/googling (add your progress to the slack message or gist/notes)
5. Did you solve it? 
  - YES: Great - add your solution to the slack channel or gist/notes and POST/UPDATE
  - NO: That's okay - now you have a well documented error and hopefully someone can help you out if you post it somewhere publicly like slack or even stackoverflow :)